v_4_0_0/filemanager.go

```diff
@@ -30,6 +30,9 @@ func RenderFiles(filesdir string, ctx interface{}) (Files, error) {
 	files := Files{}
 
 	err := filepath.Walk(filesdir, func(path string, f os.FileInfo, err error) error {
+		if err != nil {
+			return microerror.Mask(err)
+		}
```

Comment:
We shouldn't change that in already released versions.

Reply:
Just for `v_6_0_0` then?

Reply:
I'm confused. See https://gigantic.slack.com/archives/C0251MXL5/p1585566502077900